package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "importer",
```

Comment:
```suggestion
  "name": "api-config-importer",
```

Reply:
Good spot, it's done in [8796dbf](https://github.com/jembi/api-config-importer/pull/1/commits/8796dbfa02493f88ff15ce5cc85a93ec4735a0a7)